src/components/genreSelect/index.jsx

```diff
@@ -14,25 +14,24 @@ export default function GenreSelect({ genres, selectedGenre, onSelect }) {
         checkedValues.filter((checkedValue) => checkedValue !== value)
       );
     }
-    onSelect(genre);
+    onSelect(checkedValues);
   }
 
   return (
-    <div className="genre-select-container">
-      <label className="genre-select-label">Genre</label>
-      <div className={`genre-select-toggle ${expanded ? 'active' : ''}`} onClick={() => setExpanded(!expanded)}>Select Genre</div>
-      <div className="genre-options">
+    <div className={styles.container}>
+      <label className={styles.label}>Genre</label>
+      <div className={`${styles.toggle} ${expanded ? styles.active : ''}`} onClick={() => setExpanded(!expanded)} tabIndex="0">Select Genre</div>
```

Comment:
This package can help u maintaining classNames
https://www.npmjs.com/package/classnames